components/Common/Dropdown/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { DropdownItem } from '../../../types';
+import css from './index.module.scss';
+
+interface Props {
+  items: Array<DropdownItem>;
+  shouldShow: boolean;
+  styles: Object;
+}
+
+const Dropdown = ({ items, shouldShow, styles }: Props) => {
+  const mappedElements = items.map(item => {
+    const extraStyles = {
```

Comment:
Can this be one line? `const extraStyles = { fontWeight: item.active ? 'bold' : 'normal' }`

Reply:
Prettier formatting this to multiple lines